# Write a function check_positive that raises a custom exception called `NegativeValueError`
# if the input number is negative.


def check_positive(number):
    # Your code should go here.
    return ...
